student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,52 @@
-import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
-import Navbar from "../Navbar/Navbar"
-import Sidebar from "../Sidebar/Sidebar"
-import Home from "../Home/Home"
-import "./App.css"
+import * as React from "react";
+import axios from "axios";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Navbar from "../Navbar/Navbar";
+import Sidebar from "../Sidebar/Sidebar";
+import Home from "../Home/Home";
+import { useEffect, useState } from "react";
+import "./App.css";
+import SubNavBar from "../SubNavBar/SubNavBar";
+import Hero from "../Hero/Hero";
+import Logo from "../Logo/Logo";
+import ProductDetails from "../ProductDetails/ProductDetails";
 
 export default function App() {
+  const url = "http://localhost:3001";
```

Comment:
This should be defined as a constant outside of the component function, below the imports